### PR TITLE
specify chevron path

### PR DESCRIPTION
**Pull request by pcarrier**
Running in self-hosted CI workers atop ubuntu jammy, ~/.local/bin does not land in PATH even after specifying in .bashrc:

```
export PATH=$PATH:$HOME/.local/bin
```

### CODE REVIEW

File: src/scripts/chevron_output_to_file.sh [hunk 1]
```diff
@@ -10,4 +10,4 @@ set -e
 # $1 = data path
 # $2 = starting path for the output files generated by this command
 # $3 = relative path specifically that we are working on
-chevron -d "$1" "$3" > "$2/$3"
+~/.local/bin/chevron -d "$1" "$3" > "$2/$3"
```
Comment (AlexanderMann) [inline src/scripts/chevron_output_to_file.sh:13]: Are we sure that this file path is definitely where `chevron` will get installed here? https://github.com/apollographql/internal-platform-orb/pull/48/files#diff-114800ac747d171d04505053f475d59a9ab44179d9b0e0569c0eba46203ff9feR24

Comment (pcarrier) [reply src/scripts/chevron_output_to_file.sh:13]: I think `pip install --user` guarantees that?

Comment (pcarrier) [reply src/scripts/chevron_output_to_file.sh:13]:
```
        self.cmd_opts.add_option(
            "--user",
            dest="use_user_site",
            action="store_true",
            help=(
                "Install to the Python user install directory for your "
                "platform. Typically ~/.local/, or %APPDATA%\\Python on "
                "Windows. (See the Python documentation for site.USER_BASE "
                "for full details.)"
            ),
        )
```

Comment (pcarrier) [reply src/scripts/chevron_output_to_file.sh:13]:
```
site.USER_BASE
Path to the base directory for the user site-packages. Can be None if [getuserbase()](https://docs.python.org/3/library/site.html#site.getuserbase) hasn’t been called yet. Default value is ~/.local for UNIX and macOS non-framework builds, ~/Library/Python/X.Y for macOS framework builds, and %APPDATA%\Python for Windows. This value is used to compute the installation directories for scripts, data files, Python modules, etc. for the [user installation scheme](https://docs.python.org/3/library/sysconfig.html#sysconfig-user-scheme). See also [PYTHONUSERBASE](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUSERBASE).
```

Comment (AlexanderMann) [reply src/scripts/chevron_output_to_file.sh:13]: Ah, it looks like it _mostly_ guarantees that. We can actually fetch where it installed things:

- [`--user` docs](https://pip.pypa.io/en/stable/user_guide/#user-installs)
- [site.USER_BASE](https://docs.python.org/3/library/site.html#site.USER_BASE)

> Path to the base directory for the user site-packages. Can be None if [getuserbase()](https://docs.python.org/3/library/site.html#site.getuserbase) hasn’t been called yet. Default value is ~/.local for UNIX and macOS non-framework builds, ~/Library/Python/X.Y for macOS framework builds, and %APPDATA%\Python for Windows

[We can apparently get what the path is locally via:](https://docs.python.org/3/library/site.html#command-line-interface) `python -m site --user-base`

```
# alexandermann @ apollo-mannimal in ~ [9:32:22] C:127
$ python3 -m site --user-base
/Users/alexandermann/Library/Python/3.11
```

Comment (AlexanderMann) [reply src/scripts/chevron_output_to_file.sh:13]: Might be worth the future proofing while we're here?

Comment (pcarrier) [reply src/scripts/chevron_output_to_file.sh]: Done.

**Review by AlexanderMann — APPROVED** [review]

**Pull request merged**